.github/workflows/vendor_gardener.yaml

```diff
@@ -24,8 +24,8 @@ jobs:
       run: make revendor
     - name: Commit changes
       run: |
-        git config user.name github-actions
-        git config user.email github-actions@github.com
+        git config user.name gardener-robot-ci-2 
```

Comment:
Nit: Let's use `gardener-robot-ci-1`. You could also trim the trailing whitespace.